Northstar.CustomServers/mod/scripts/vscripts/mp/_base_gametype_mp.gnut

```diff
@@ -630,6 +632,27 @@ void function SetRecalculateRespawnAsTitanStartPointCallback( entity functionref
 	file.recalculateRespawnAsTitanStartPointCallback = callbackFunc
 }
 
+bool function ClientCommandCallbackChangeTeam( entity player, array<string> args )
+{
+	if ( !( player in file.playerChangeTeamTimeBuffer ) )
+		file.playerChangeTeamTimeBuffer[ player ] <- Time() + 5
+	else
+	{
+		if ( file.playerChangeTeamTimeBuffer[ player ] > Time() )
+		{
+			SendHudMessage( player, "Team Switching is on Cooldown", -1, 0.4, 255, 255, 255, 255, 0.15, 3.0, 0.5 )
+			return true
+		}
+	}
```

Comment:
Would be great if we didn't mix indent and normal scopes

Reply:
Will change then